### ISSUE TITLE
For optional property, question mark inside key quote instead of outside

### ISSUE THREAD

**Issue by D1no** [MariusAlch/json-to-ts#1]
When generating definition files with json-to-ts (either in vs code or directly via the npm module), it puts the question mark for an optional key right inside the key string. Instead of outside of it.

See:
```typescript
{
'features-CC_LMIV_25-Fanggerätkategorie_tlcm?': string[];
                                   // (..) m?': (..) should be (..) m'?: (..)
}
```

Shouldn't it be:
```typescript
'features-CC_LMIV_25-Fanggerätkategorie_tlcm'?: string[];
```

**Comment by MariusAlch** [comment]
It should thanks for issue I will try to fix that today.

**Comment by MariusAlch** [comment]
Fixed with 1.4.3 release. Closing this issue. Again thanks for the issue.

**Comment by D1no** [comment]
Thanks!
